### PR TITLE
testing external libs

### PR DESCRIPTION
**Pull request by sangallimarco**
attempt to split code

### CODE REVIEW

File: src/components/profile/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import {h, Component} from 'preact';
 import style from './style';
 import {Streams} from './streams';
-import {http, StreamManager} from '../../libs';
+import {http} from 'barbarojs-http';
```
Comment (sangallimarco) [inline src/components/profile/index.js]: bellezza

File: webpack.config.babel.js [hunk 2]
```diff
@@ -55,6 +53,10 @@ module.exports = {
 				exclude: /node_modules/,
 				loader: 'babel'
 			}, {
+				test: /\.jsx?$/,
+				include: /node_modules\/barbarojs/,
```
Comment (sangallimarco) [inline webpack.config.babel.js]: awsomenessssss

File: package.json [hunk 3]
```diff
@@ -79,6 +79,8 @@
     "webpack-dev-server": "^1.15.0"
   },
   "dependencies": {
+    "barbarojs-http": "git+https://github.com/barbarojs/barbarojs-http.git",
```
Comment (MatteoWebDesigner) [inline package.json]: I thought you were using npm

Comment (sangallimarco) [inline package.json]: I need to take a look how to publish

Comment (sangallimarco) [inline package.json]: need to have a look how to publish... but we need to check if it is ok first

**Pull request merged**